src/main/java/ru/itis/services/WishListService.java

```diff
@@ -21,13 +21,16 @@ public WishListService(WishListRepository wishListRepository, UserService userSe
     }
 
     public WishList addNewWishList(String title, String token) {
-        User currentUser = userService.findUserByToken(token).get();
-        WishList newWL = WishList.builder()
-                .title(title)
-                .author(currentUser)
-                .build();
-        currentUser.getWishLists().add(newWL);
-        return wishListRepository.save(newWL);
+        if (userService.findUserByToken(token).isPresent()) {
```

Comment:
It's too procedural.

Reply:
suggestions?

Reply:
- What does `currentUser.getWishLists().add(newWL);` do?
- `return null;` 😢

Reply:
Why do you query database twice? `userService.findUserByToken(token)`

Reply:
got it, fixed